packages/migrations/src/createStakerNetworkAndConnectStakerPkgs.ts

```diff
@@ -0,0 +1,39 @@
+import { docker } from "@dappnode/dockerapi";
+import { params } from "@dappnode/params";
+import { logs } from "@dappnode/logger";
+import { ensureStakerPkgsNetworkConfig } from "@dappnode/stakers";
+
+/**
+ * Creates the staker network and connects the staker packages to it
+ */
+export async function createStakerNetworkAndConnectStakerPkgs(): Promise<void> {
+  await createDockerStakerNetwork();
+  await ensureStakerPkgsNetworkConfig();
+}
+
+/**
+ * Creates the docker staker network
+ */
+async function createDockerStakerNetwork(): Promise<void> {
+  try {
+    const stakerNetwork = docker.getNetwork(params.DOCKER_STAKER_NETWORK_NAME);
+    await stakerNetwork.inspect();
+    logs.info(`docker network ${params.DOCKER_STAKER_NETWORK_NAME} exists`);
+  } catch (e) {
+    if (e.statusCode === 404) {
+      logs.info(`Creating docker network ${params.DOCKER_STAKER_NETWORK_NAME}`);
```

Comment:
I would maybe add "Docker network `params.DOCKER_STAKER_NETWORK_NAME` not found, creating it" as a debug log to add more context.